charts/monitoring/values.yaml

```diff
@@ -12,11 +12,12 @@ grafanaIngress:
 prometheusIngress:
   annotations: {}
 
+# Adds the CulsterRole needed for prometheus to the deploment. See clusterrole.yaml
+adminDeploy: false
```

Comment:
Hmm, should this be true or false by default?

I think there are other parts of this chart that assume admin permissions - the operators for instance. So I think the default "role" of this chart is admin, meaning this should be true.

Reply:
Also, since adminDeploy is true, we should configure the prometheus server to use this role by default

prometheus:
    server:
      useExistingClusterRoleName: prometheus-admin-cluster-role
      namespaces:
        - {{ namespace conditional }}